pkgs/development/tools/go-packr/default.nix

```
@@ -0,0 +1,23 @@
+{ stdenv, buildGoPackage, fetchgit }:
+
+buildGoPackage rec {
+  name = "go-packr-${version}";
+  version = "v1.13.7";
```

Please update to the latest stable version currently "2.7.1".
```suggestion
  version = "2.7.1";
```